deploy-manage/deploy/elastic-cloud/create-an-organization.md

```diff
@@ -76,6 +76,13 @@ For more information, check the [{{ech}} documentation](cloud-hosted.md).
 * Scaling is limited for {{serverless-short}} projects in trials. Failures might occur if the workload requires memory or compute beyond what the above search power and search boost window setting limits can provide.
 * We monitor token usage per account for the Elastic Managed LLM. If an account uses over one million tokens in 24 hours, we will inform you and then disable access to the LLM. This is in accordance with our fair use policy for trials. 
 
+**Inference tokens**
```

Comment:
having this be a sibling to hosted/serverless is not ideal because they are "sibling" deployment types. we should add some applies tags to indicate that the inference tokens limitation applies to both deployment types

```suggestion
**Inference tokens** {applies_to}`ess: ga` {applies_to}`serverless: ga`
```